code/obj/item/device/studio_monitor.dm

```diff
@@ -204,7 +211,10 @@
 			var/mob/living/carbon/human/virtual/V = target
 			. = istype(V.ears, /obj/item/device/radio/headset/syndicate) || istype(V.head, /obj/item/clothing/head/helmet/space/syndicate)
 		else
-			. = istype(target.ears, /obj/item/device/radio/headset/syndicate)
+			if(is_syndicate)
+				. = istype(target.ears, /obj/item/device/radio/headset/syndicate)
+			else
+				. = istype(target.ears, /obj/item/device/radio/headset) //Nanotrasen guitar, Nanotrasen tunes
```

Comment:
don't you mean like a ntso or captain or something headset

Reply:
Wasn't sure what it should be set to for a while, settled on a command headset (which also includes NTSOs and such)